src/dotnet/xlf/CommonLocalizableStrings.pt-BR.xlf

```diff
@@ -612,33 +612,33 @@
       </trans-unit>
       <trans-unit id="UnsupportedProjectType">
         <source>Unsupported project type. Please check with your sdk provider.</source>
-        <target state="new">Unsupported project type. Please check with your sdk provider.</target>
-        <note></note>
+        <target state="translated">Tipo de projeto sem suporte. Verifique com seu provedor de SDK.</target>
+        <note />
       </trans-unit>
       <trans-unit id="InvalidSolutionFormatString">
         <source>Invalid solution `{0}`. {1}</source>
-        <target state="new">Invalid solution `{0}`. {1}</target>
-        <note></note>
+        <target state="translated">Solução inválida `{0}`. {1}</target>
+        <note />
       </trans-unit>
       <trans-unit id="ArgumentsProjectDescription">
         <source>The project file to operate on. If a file is not specified, the command will search the current directory for one.</source>
-        <target state="new">The project file to operate on. If a file is not specified, the command will search the current directory for one.</target>
-        <note></note>
+        <target state="translated">O arquivo de projeto a ser operado. Se um arquivo não for especificado, o comando pesquisará um no diretório atual.</target>
+        <note />
       </trans-unit>
       <trans-unit id="ArgumentsSolutionDescription">
         <source>Solution file to operate on. If not specified, the command will search the current directory for one.</source>
-        <target state="new">Solution file to operate on. If not specified, the command will search the current directory for one.</target>
-        <note></note>
+        <target state="translated">O arquivo de solução a ser operado. Se não for especificado, o comando pesquisará um no diretório atual.</target>
```

Comment:
O arquivo the solução alvo. Se um arquivo não for especificado, o comando procurará por um no diretório atual.